edi_backend_partner/models/res_partner.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    edi_backend_ids = fields.One2many(
+        comodel_name="edi.backend", inverse_name="partner_id",
+    )
+    edi_backend_count = fields.Integer(compute="_compute_edi_backend_count")
+
+    def _compute_edi_backend_count(self):
+        for rec in self:
```

Comment:
if you expect this to run a large recordset it would be better to use read_group

Reply:
actually this field is only used in the form view, so it will be computed one by one.